_sample_posts/2016-08-16-Measuring-software-engineering-competency.md

```diff
@@ -72,40 +74,52 @@ Now let's elaborate a little bit on the items from this list.
+We need to track what's going on on the project and guarantee that defects or tasks won't be forgotten or overlooked. It doesn’t matter what it is - whiteboard with stickers on it, online tools or something else, but we have to use something to keep things organised and the whole process more transparent for everybody. 
 
 ## Do you have daily status check-in meetings?
 
-Everyone in team should know what's going on in project to avoid miscommunication issues and distractions.
+Communication is important, keep everybody in the loop but do not bore them with details. 
 
 ## Do you have short iterational process?
 
-Short process, like weeks or two, keeps team focused on a small goals  which is always good for a quality.  
+Short process, like weeks or two, keeps team focused on a small and achievable goals.
 
-## Do you have unit tests with at least 50% coverage?
+## Do you have unit tests?
 
-Unit testing is important for a code quality because it helps developers to build more flexible code design and update/refactor the code without fear to break existing functionality. It's not necessary to get crazy and always keep code coverage 100% but code should be covered by unit tests at least in half.  
+We don't know the better way to make sure that code does what it's supposed to, then to have another code that runs it and check results.
 
 ## Do you have automated end-to-end tests in DSL?
 
+Unit tests is about to keep *code quality*, but they do little help with the overall *application quality*. It doesn't matter how good is your code if application doesn't do what it supposed to do. Creating end-to-end tests in DSL achieve at least two advantages: you will be sure that application behave as expected and DSL will give you human-friendly documentation about high-level logic that cover your particular domain. It's not, actually, necessary to involve the customer or the tester to process of creating these tests, but it will be much easier to explain application behaviour and limitations to them when you have it.
+
 ## Do you have an Acceptance Person?
 
+Job is done, when Man Who Accepts The Job are said it to. You should have person for that, and it's not necessary it should be customer or owner in a terms of SCRUM (which is the best option here) - it could be QA or other developer in your team, but not who participated to the particular feature implementation.
+
 ## Do you do code reviews?
 
+Code review is extremely useful technique for knowledge sharing and team education and application quality support, [if you are doing it right](http://www.savvyclutch.com/Make-Code-Review-Useful-Again/). It’s impossible to keep team code and engineering decisions understandable, organized and with a minimum architecture issues without code review.
```

Comment:
is an extremely useful technique for knowledge sharing, team education, and application quality support